facebook_connector.py

```diff
@@ -45,8 +53,23 @@ def resolve_user_id(user_id):
     Returns:
         The name of the user if it is able to find it, otherwise the input
     """
-    # TODO (neitsch): This method is not implemented yet.  Use the user_id
-    # (format: 12345678@facebook.com) to get the actual name of the user
+    fb_id_pattern_match = _FB_ID_PATTERN.match(user_id)
```

Comment:
This method doesn't quite take a "user_id" it takes the full URL, perhaps have it accept both?  Or clarify the pydoc.